source/CMakeLists.txt

```diff
@@ -88,7 +102,7 @@ write_basic_package_version_file(
 )
 
 # create config file
-configure_package_config_file(${CMAKE_CURRENT_SOURCE_DIR}/Config.cmake.in
+configure_package_config_file(${CMAKE_CURRENT_SOURCE_DIR}/control_libraries-config.cmake.in
   "${CMAKE_CURRENT_BINARY_DIR}/${PROJECT_NAME}Config.cmake"
```

Comment:
The base config input file was renamed, but the generated output is still `${PROJECT_NAME}Config.cmake`. Should it also be changed to the lowercase naming convention `<lowercasePackageName>-config.cmake`? It shouldn't be a breaking change since `find_package` looks for both name types but I would also make sure that's tested before confirming it.

Reply:
Hm yeah good point, I think I like `${PROJECT_NAME}Config.cmake` better. I renamed the file